atomic_reactor/plugins/pre_check_base_image.py

```diff
@@ -95,13 +88,10 @@ def run(self):
 
             image_with_digest = self._get_image_with_digest(image)
             if image_with_digest is None:
-                self.log.warning("Cannot resolve manifest digest for image '%s'", image)
-            else:
-                self.log.info("Replacing image '%s' with '%s'", image, image_with_digest)
-                image = image_with_digest
+                raise RuntimeError("Cannot resolve manifest digest for image {}".format(image))
+
+            self.log.info("Replacing image '%s' with '%s'", image, image_with_digest)
```

Comment:
this :                 image = image_with_digest
was replacing that image via: self.workflow.dockerfile_images[parent] = image

but now you are only logging replacing image but doing not :)

Reply:
fixed.